app/controllers/builds_controller.rb

```diff
@@ -131,6 +131,27 @@ def modified_time
     end
   end
 
+  def updated_parts
```

Comment:
I am not liking the name `updated_parts`. Perhaps `refresh_build_part_info` ?

Reply:
ill change